src/index.tsx

```diff
@@ -185,21 +185,17 @@ export class App extends React.Component<{}, {
     }
 
     render() {
-        // fetch contents so we can get possible values from the contents . 
-        return <PluginShell title="PinColor" details="Set custom pin colors">
-            <SheetContainer
-                onReady={this.renderBody1}
-                fetchContents={true}
-                requireTop={true}>
-            </SheetContainer>
+        // fetch contents so we can get possible values from the contents .
+        return <PluginShell title="PinColor" description="Set custom pin colors">
+            {this.renderBody1()}
         </PluginShell>
 
     };
 }
 
 ReactDOM.render(
-    <div>
-        <App></App>
-    </div>,
+    <SheetContainer fetchContents={true} requireTop={true}>
```

Comment:
PluginsShell's job is to provide the context to the children, and it's generally best practice to wrap the context provider component at app initialization like we are doing there.

- In case of multiple sheets per page, the solution is to use another sheet container, let's say `<SheetsContainer>` or `<MultipleSheetsContainer>`, and implement the low level API. I don't think we want to have a monolith `<SheetContainer>` component that handles every possible case